practice_app/pages/index.js

```diff
@@ -1,9 +1,13 @@
 import { useState } from 'react'
```

Comment:
Currently map is responding only once. This needs to be fixed.

Reply:
It is because missing api key for positionstack. If you create .env file in project directory and paste 

POSITION_STACK_API_KEY=ae7fc0efbe65c249d2ca580ec51b3fdd

into that file, it will work finely

Reply:
I did that and it does work fine but i am just curious about how the instructors are going to be aware of this or will you give them this file somehow?

Reply:
I am not sure, I think API keys should not be pushed to github. But perhaps we share the API keys over slack so that whenever someone wants to test they can put them in an `.env` file and run the app?

Reply:
I actually have no idea what to do about this. Sharing the key over Slack would make our group be able to run the app but will we share it with the instructors aswell and if we will, then how are we going to do it?

Reply:
When deploying it I believe we can pass the environment variables (api key) to Docker somehow, so it won't be a problem when demo-ing